### PR TITLE
[Snap]: Implementation of navbar dropdown (network)

### PR DESCRIPTION
**Pull request by AnthonyLaw**
## What was the issue?
- Implement navbar network dropdown 

## What's the fix?
- Backend
  - Added fetch data utils.
  - Added statistic client which is used to query network nodes
  - Added switch network method
  - Added `getNetowork` and `switchNetwork`, refactor snap onRpcRequest,

- Frontend 
  - Implement on click network dropdown switch network

### CODE REVIEW

File: snap/backend/src/config.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const config = {
+	statisticsServiceURL: {
+		mainnet: 'https://symbol.services',
+		testnet: 'https://testnet.symbol.services'
```
Comment (Wayonb) [inline snap/backend/src/config.js:4]: does nodewatch works for this? - https://nodewatch.symbol.tools/api/symbol/nodes/peer

File: snap/backend/src/utils/networkUtils.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import statisticsClient from '../services/statisticsClient.js';
+import stateManager from '../stateManager.js';
+
+const networkUtils = {
+	async switchNetwork({ state, requestParams }) {
```
Comment (Wayonb) [inline snap/backend/src/utils/networkUtils.js:5]: Can the switchNetwork() be used in the index.js to initialize the default mainnet?

File: snap/backend/test/services/statisticsClient_spec.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+jest.spyOn(fetchUtils, 'fetchData').mockImplementation();
+
+describe('statisticsClient', () => {
+	beforeEach(() => {
+		jest.clearAllMocks();
+	});
+
+	describe('getNodeInfo', () => {
+		it('should fetch node info successfully', async () => {
```
Comment (Wayonb) [inline snap/backend/test/services/statisticsClient_spec.js:13]: can fetch node info successfully?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: snap/backend/src/utils/networkUtils.js (hunk 2 above)
Comment (AnthonyLaw) [reply snap/backend/src/utils/networkUtils.js:5]: yes, you are right, I will apply for that.

File: snap/backend/src/config.js (hunk 1 above)
Comment (Wayonb) [reply snap/backend/src/config.js:4]: It's not a requirement for this PR, but something you should think about.  Any feature lacking from NodeWatch please post in discord for us to improve on 🙏🏾

Comment (AnthonyLaw) [reply snap/backend/src/config.js:4]: Sure 👍🏼

**Comment by AnthonyLaw** [comment]
@Wayonb I did make a small refactor in https://github.com/symbol/product/pull/929/commits/470c536e80b84d7d4be2e11c4277f6891ad86074

I added `initialSnap` method, which will set up the network.

**Review by Wayonb — APPROVED** [review]
👍🏾

**Pull request merged**